core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2679,6 +2688,10 @@ object SparkContext extends Logging {
         if (threadCount <= 0) {
           throw new SparkException(s"Asked to run locally with $threadCount threads")
         }
+        if (cpusPerTask > threadCount) {
+          throw new SparkException(s"$threadCount threads available when run local[$threads], " +
```

Comment:
Here and below I don't think "local[...]" adds much beyond threadCount in the message

Reply:
Please pardon me as my English isn't very good.
I do not understand this comment.

Reply:
You already report threadCount in the message; local[threads] doesn't add information. It can be removed

Reply:
Sometimes `threadCount` is not the same to  `threads` such as `local[*]`